ts/data/project.ts

```diff
@@ -166,14 +166,17 @@ export const projectDefinition: TableDefinition<ProjectSummary, void> = {
 };
 
 const completedFilterKey = "COMPLETED";
+const tableFilterKey = "TABLE";
 
 function displayFilteredProject(
   count: number,
   projectName: string,
   filterValue: string
 ) {
-  var url = new URL(document.location.href);
-  var params = url.searchParams;
+  const params = new URLSearchParams();
+  const tableFilterValue =
+    filterValue === "Informatics Review" ? filterValue : filterValue + "s";
```

Comment:
Please change the tab name to "Informatics Reviews" instead of making this exception. That way they're all consistently plural

Reply:
Will do!